dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/intermediate/components/Utils.kt

```diff
@@ -20,3 +22,27 @@ fun SectionConfigBuilder.addStandardCellWithValueGetterFactory(
         valueGetter = valueGetter,
     )
 }
+
+/**
+ * Add a cell to the section with configuration shared between components
+ * and a component-specific value-getter
+ */
+fun SectionUploadConfigBuilder.addStandardCellWithValueGetterFactory(
```

Comment:
The name of the function is misleading. There are no value getter factories in the upload page generation.

Reply:
Should be smth like `addField`.  Because in the upload config the terminology is "field, subcategory, category" instead of "cell" and "section".

Reply:
Fixed